.gitignore

```diff
@@ -0,0 +1,12 @@
+.bundle/
+log/*.log
+pkg/
+test/dummy/db/*.sqlite3
+test/dummy/log/*.log
+test/dummy/tmp/
+test/dummy/.sass-cache
+locale/*.mo
+locale/*/*.edit.po
+locale/*/*.po.time_stamp
+locale/*/*.pox
+.rstrip
```

Comment:
Wanna add some of theses?
https://github.com/MiamiOH/foreman_custom_tab/blob/master/.gitignore